packages/kit/src/exports/node/index.js

```diff
@@ -62,14 +61,12 @@ function get_raw_body(req, body_size_limit) {
 				size += chunk.length;
 				if (size > length) {
 					cancelled = true;
-					controller.error(
-						error(
-							413,
-							`request body size exceeded ${
-								content_length ? "'content-length'" : 'BODY_SIZE_LIMIT'
-							} of ${length}`
-						)
-					);
+					controller.error({
+						status: 413,
+						message: `request body size exceeded ${
+							content_length ? "'content-length'" : 'BODY_SIZE_LIMIT'
+						} of ${length}`
+					});
```

Comment:
somewhat of a drive-by-fix - the "`error` throws itself" change caused a breakage here. It's not really necessary to use the `error` function in this context, so I replaced both usages with POJO in this file